src/Components/Modals/CreateActivationKeyWizard.js

```diff
@@ -91,11 +81,28 @@ const CreateActivationKeyWizard = ({
   const steps = [
     {
       id: 0,
-      name: 'Name',
+      name: 'Name and Description',
       component: (
-        <SetNamePage name={name} setName={setName} nameIsValid={nameIsValid} />
+        <div className="pf-l-grid pf-m-gutter">
+          <div className="pf-v5-u-mb-xl">
+            <SetNamePage
+              name={name}
+              setName={setName}
+              nameIsValid={nameIsValid}
+            />
+          </div>
+          <div className="pf-v5-u-mb-xl">
+            <div className="pf-v6-u-text-wrap">
+              <SetDescriptionPage
+                description={description}
+                setDescription={setDescription}
+                descriptionIsValid={descriptionIsValid}
+              />
+            </div>
+          </div>
+        </div>
```

Comment:
Suggestion: This goes with the comment about `setDescriptionPage`, but I think this step should have a standalone component it passes in, instead of all this inline js.

Something like `component: <SetNameAndDescriptionPage {...whatever props} />`